pkg/action/install.go

```diff
@@ -379,7 +379,9 @@ func (i *Install) performInstall(c chan<- resultMessage, rel *release.Release, t
 		}
 	}
 
-	if i.Wait {
+	waitBeforePostHooks := !i.DisableHooks && i.cfg.hasPostInstallHooks(rel) || i.cfg.hasPostUpgradeHooks(rel)
```

Comment:
Do we need to consider an option to allow `post-install|upgrade` (and `rollback` below) hooks run before waiting? To avoid a breaking users who might depend on this behavior here?

Reply:
I don't see how this could break users. The behavior was always inconsistent. If they test their charts, they would have discovered the failure.

All the CD systems either use wait or they template everything and use their own applicator that applies the post-* jobs after the main body is ready.

I don't think polluting the cli with yet-another-flag has any value here.